cmd/mimir/config-descriptor.json

```diff
@@ -2116,20 +2116,42 @@
               "kind": "field",
               "name": "failure_threshold",
               "required": false,
-              "desc": "Max number of requests that can fail in a row before the circuit breaker opens",
+              "desc": "Max percentage of requests that can fail over period before the circuit breaker opens",
               "fieldValue": null,
               "fieldDefaultValue": 10,
               "fieldFlag": "ingester.client.circuit-breaker.failure-threshold",
               "fieldType": "int",
               "fieldCategory": "experimental"
             },
+            {
+              "kind": "field",
+              "name": "failure_execution_threshold",
+              "required": false,
+              "desc": "How many requests must have been executed in period for the circuit breaker to be eligible to open for the rate of failures",
+              "fieldValue": null,
+              "fieldDefaultValue": 100,
+              "fieldFlag": "ingester.client.circuit-breaker.failure-execution-threshold",
+              "fieldType": "int",
+              "fieldCategory": "experimental"
+            },
+            {
+              "kind": "field",
+              "name": "period",
+              "required": false,
+              "desc": "Moving window of time that the percentage of failed requests is computed over",
+              "fieldValue": null,
+              "fieldDefaultValue": 10000000000,
```

Comment:
Based on offline discussion, I think you are right @jhalterman that a longer period here makes such that the default value works for both read and write paths. Looking at a handful of production cells, the lowest traffic one was doing ~15 `/cortex.Ingester/QueryStream` requests / sec / ingester. Changing this period to `1m` would ensure that all cells meet the "100 requests" threshold for circuit breaking on the read path.